pyquil/api/_abstract_compiler.py

```diff
@@ -55,9 +55,6 @@ class EncryptedProgram:
     ro_sources: dict[MemoryReference, str]
     """Readout sources, mapped by memory reference."""
 
-    recalculation_table: list[str]
-    """A mapping from memory references to the original gate arithmetic."""
-
```

Comment:
This is a breaking change. To avoid a major version bump or breaks, we could also deprecate the field.